00-HTML-CSS-basics/excercise 2.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html>
+<head>
+   <link href="normalize.css" rel="stylesheet">
+   <link href="style.css" rel="stylesheet">
+   <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+   <meta charset="UTF-8">
+   <title>Excercise 2</title>
+</head>
+
+<body class="body">
+
+    <div class="top-header-container"><header class="top-header" class="header"> <p> header </p> </header></div>
+
+    <div class="middle-elements">
+
+    <div class="leftside-nav-containers"> <nav class="leftside-nav" class="navigation"> <a href=""> <p> nav </p> </a> </nav> </div>
+
+    <div class="middle-elements-containers"> <section class="middle-elements-section" class="content"> <p> section </p> </section>
```

Comment:
Try tou use one line per tag, this could be:

```
<div class="middle-elements-containers"> 
  <section class="middle-elements-section" class="content"> 
    <p> section </p> 
  </section>
  <header class="middle-elements-header" class="header">
    <p> header </p>
  </header>
  <article class="middle-elements-article" class="article">
    <p> article </p> 
  </article>
  <footer class="middle-elements-footer" class="footer">
    <p> footer </p>
  </footer>
</div>
```